app/services/foreman_ansible/roles_importer.rb

```diff
@@ -1,24 +1,55 @@
 module ForemanAnsible
-  # Service to import roles from the filesystem.
-  # Should be extracted to a gem that both the proxy and Foreman can use.
+  # imports roles from smart proxy
```

Comment:
Just curious, why would we encourage people to handle local fs? We support it only through proxy for puppet modules. Having two paths that we have to support seems as extra burden to me.